samples/CMakeLists.txt

```diff
@@ -223,7 +223,9 @@ target_sources(gltf-resources PRIVATE ${RESGEN_SOURCE})
 # Common library
 # ==================================================================================================
 
-set(APP_LIBS filament sdl2 stb math filamat utils getopt imgui filagui image camutils common-resources)
+# We include gltfio in all desktop samples even if they don't use GLTF because the IBL loader needs STB.
+
+set(APP_LIBS filament sdl2 stb math filamat utils getopt imgui filagui image camutils gltfio common-resources)
```

Comment:
It looks like we already link against `stb`, why do we also need to link against `gltfio` now?

Reply:
Yeah this is confusing: stb is actually a header-only dependency.

Here I removed the copy of `Image.cpp` from our samples directory (which brought in the STB impl) because it caused a doubly-defined link error with gltfio (which already links against STB).

Reply:
It is unfortunate to force all samples to link against gltfio though. Couldn't non-glTF samples link against an implementation of STB that's not in gltfio? Should we just switch the samples to `libimage`?

Reply:
Fixed by adding a commit that brings back `Image.cpp`, this was the reason it existed.  It was a mistake to completely remove it from all samples.